tests/grate/asm/fs_vs_tests.txt

```diff
@@ -0,0 +1,13 @@
+alu_buffer_size = 1
+pseq_to_dw_exec_nb = 1
+
+.asm
+EXEC
+	MFU:	var unk(0x104E51BA0) r0.fp20, r0.fp20, r0.fp20, r0.fp20
```

Comment:
It actually seems to be instructions (and use the RCP-instruction from the MFU unit for the division), but using some special register outputs for the weights.

My hope is that this can be used to implement things like flat-interpolation, for instance.